quinn-proto/src/lib.rs

```diff
@@ -282,7 +281,7 @@ pub struct Transmit {
     /// Explicit congestion notification bits to set on the packet
     pub ecn: Option<EcnCodepoint>,
     /// Contents of the datagram
-    pub contents: Bytes,
+    pub size: usize,
```

Comment:
Outdated comment. Should now say something like "Amount of data written to the caller-supplied buffer".